reference/constraints/CardScheme.rst

```diff
@@ -119,6 +119,7 @@ Valid values are:
 * ``LASER``
 * ``MAESTRO``
 * ``MASTERCARD``
+* ``MIR``
```

Comment:
In addition to adding this line, please update as follows the `versionadded:: 4.3` displayed below this list. Thanks

```rst
.. versionadded:: 4.3

    The ``UATP`` and ``MIR`` number schemes were introduced in Symfony 4.3.
```